metaflow/plugins/metadata/local.py

```diff
@@ -173,7 +208,11 @@ def _makedirs(path):
                 raise
 
     def _ensure_meta(
-            self, obj_type, run_id, step_name, task_id, tags=[], sys_tags=[]):
+            self, obj_type, run_id, step_name, task_id, tags=None, sys_tags=None):
+        if tags is None:
+            tags = []
```

Comment:
since tags are sets, we could make them sets everywhere for clarity, i.e. `tags = set()` here. We need to convert sets to lists only for JSON encoding